Jenkinsfile.deploy

```diff
@@ -89,7 +76,25 @@ node('multiarch-' + env.BASHBREW_ARCH) { ansiColor('xterm') {
 			sh """#!/usr/bin/env bash
 				set -Eeuo pipefail
 
-				${ shell }
+				commands=( ${ shell } )
```

Comment:
Won't this split the expanded Groovy variable `shell` on whitespace?

Reply:
Yup. That's why it does `crane_deploy_commands | @sh` to create it. The function will return a stream of crane commands and then each one will be shell quoted to be a single shell string.

```
crane_deploy_commands
crane copy ...
crane copy ...
-> @sh
'crane copy ...'
'crane copy ...'
```

Reply:
And to be clear, this won't be shell doing expansion/splitting, but rather the Groovy will embed the generated and appropriately quoted shell code directly in the string it passes to the shell for invocation, so the end result of what the shell actually executes will be literally something like:

```bash
commands=( 'crane copy ...'
'crane copy ...' )
```